wplogin

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+wpinstall=`dirname $(wp config path 2>/dev/null) 2>/dev/null`
```

Comment:
```suggestion
wpinstall=`dirname $(wp --skip-plugins --skip-themes config path 2>/dev/null) 2>/dev/null`
```

Reply:
This looks great, I am going to recommend that you use the skip plugins and themes flags to speed up the script and to support crappy themes and plugins that break wpcli.